common/app/mvt/MultiVariateTesting.scala

```diff
@@ -39,6 +39,17 @@ object ABNewRecipeDesign extends TestDefinition(
   }
 }
 
+object ABNavigationMoreToggle extends TestDefinition(
+  name = "ab-navigation-more-toggle",
+  description = "Users in the test will see a more link in the navigation subnav",
+  owners = Seq(Owner.withGithub("gustavpursche")),
+  sellByDate = new LocalDate(2017, 4, 21)
```

Comment:
You are right - talked with Stephan in the morning about it. We'll run it till Monday.